### DISCUSSION TITLE
Movie file stops being read

### DISCUSSION THREAD

**Discussion by lauraangeja** [bonsai-rx/bonsai#1349] [Q&A]
Hi! 

I am experiencing a weird error which is my movie files are stopping to run at about half way through and I don't really understand why is this.

 I don't really think it is a Bonsai problem because i just ran a script with only a FileCapture node and they stop playing too.

I would be very grateful if someone knew or has any idea why this is happening and how to fix this because I have checked the properties and everything I could think of but all my movie test files stop being analysed at about half way.

I attach here one of my file of open fied test that are having this issue.
https://drive.google.com/file/d/1OibZ9dGqQo6TnBVDR-AHXG0y7vicEw-L/view?usp=share_link

**Comment by bruno-f-cruz** [comment]
Not sure what is going on. I have tested the attached video on my PC and it seems to play all video (the same number of frames as VLC player does). Can you share your workflow?

**Comment by lauraangeja** [comment]
Hm that's strange. To test if it was my workflow that was creating the problem, I just created a new one with the FileCapture node and it stop reproducing the video at about frame 31 000 (about half). It simply shuts down.
